documentation/tutorials/02-error-handling.md

```diff
@@ -220,34 +281,52 @@ iex -S mix
 ```
 
 ```elixir
-# Test with a valid @example.com email (should succeed)
+# ✅ SUCCESS: Normal email succeeds
 {:ok, user} = Reactor.run(ResilientUserRegistration, %{
   email: "alice@example.com",
   password: "secretpassword123"
 })
 
-# Test with a non-@example.com email (will trigger retry logic)
+# 🔄 RETRY: Network timeout triggers retry logic
```

Comment:
This returns `{:error, reasons}` because it fails the retries as well, right? Might be worth mentioning.

Reply:
Ok, never mind. Should have read it til the end first :sweat_smile: